README.md

```diff
@@ -1,44 +1,50 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# :construction: IAPPS Web App Front-End
 
-## Available Scripts
+This repo contains the setup for IAPPS Frontend.
 
-In the project directory, you can run:
+## DO NOT ADD ANY EXTERNAL LIBRARIES
 
-### `npm start`
+Everything needed to develop the project has been setup properly including routing, css libraries, liniting files, build process. DO NOT add any external library without discussion. Please simply take a pull from develop and start adding HTML, CSS and TypeScript Code.
```

Comment:
Add details on the git flow as well. Simply taking a pull from develop is not enough. There should a complete cycle of feature branch, testing, merge, testing and then deployment.

Reply:
If you can, I'd love it if you can define the gitflow. Since you are more experienced in Git and code management than I am.

Reply:
https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow